base_local_planner/src/map_grid.cpp

```diff
@@ -265,7 +268,12 @@ namespace base_local_planner{
     }
     if (!started_path) {
       ROS_ERROR("None of the points of the global plan were in the local costmap, global plan points too far from robot");
-      return;
+      return mbf_msgs::ExePathResult::OUT_OF_MAP;
+    }
+
+    if (local_goal_in_obstacle) {
+      ROS_WARN("None of the points of the global plan sufficiently far away from the start were in free space");
+      return mbf_msgs::ExePathResult::BLOCKED_PATH;
```

Comment:
S: the MBF PRs is approved; maybe we can merge it, update the sm and change here to BLOCKED_GOAL?

Reply:
yes that's the plan (as written in the description)